lib/kernel/doc/src/inet.xml

```diff
@@ -1566,18 +1582,88 @@ setcap cap_sys_admin,cap_sys_ptrace,cap_dac_read_search+epi beam.smp</code>
 	      option.
 	    </p>
           </item>
-          <tag><c>{reuseaddr, Boolean}</c></tag>
+          <tag><c>{reuseaddr, Boolean}</c><marker id="option-reuseaddr"/></tag>
+          <item>
+            <p>
+              Allows or disallows reuse of local address. By default, reuse
+              is disallowed.
+            </p>
+            <note>
+              <p>
+                On windows <c>{reuseaddr, true}</c> will have no effect unless
+                also <seeerl marker="#option-reuseport"><c>{reuseport,
+                true}</c></seeerl> is set. If both are set, the
```

Comment:
...is also set...

Reply:
there is in also before the option which I guess is correct